examples/sampling/graphbolt/node_classification.py

```diff
@@ -0,0 +1,331 @@
+    # [Input]:
+    # 'features': The node features.
+    # 'node_feature_keys': The node feature keys (list) to be fetched.
+    # [Output]:
+    # A FeatureFetcher object to fetch node features.
+    # [Role]:
+    # Initialize a feature fetcher for fetching features of the sampled
+    # subgraphs.
+    ############################################################################
+    datapipe = datapipe.fetch_feature(features, node_feature_keys=["feat"])
+
+    ############################################################################
+    # [Step-5]:
+    # self.copy_to()
+    # [Input]:
+    # 'device': The device to copy the data to.
+    # [Output]:
+    # A CopyTo object to copy the data to the specified device.
+    ############################################################################
+    datapipe = datapipe.copy_to(device=args.device)
+
+    ############################################################################
+    # [Step-6]:
+    # gb.MultiProcessDataLoader()
+    # [Input]:
+    # 'datapipe': The datapipe object to be used for data loading.
+    # 'args.num_workers': The number of processes to be used for data loading.
+    # [Output]:
+    # A MultiProcessDataLoader object to handle data loading.
+    # [Role]:
+    # Initialize a multi-process dataloader to load the data in parallel.
+    ############################################################################
+    dataloader = gb.MultiProcessDataLoader(
+        datapipe, num_workers=args.num_workers
+    )
+
+    # Return the fully-initialized DataLoader object.
+    return dataloader
+
+
+@torch.no_grad()
+def evaluate(args, model, graph, features, itemset, num_classes):
+    model.eval()
+    ys = []
+    y_hats = []
+    dataloader = create_dataloader(
+        args, graph, features, itemset, is_train=False
+    )
+
+    for step, data in tqdm.tqdm(enumerate(dataloader)):
+        blocks = data.to_dgl_blocks()
+
+        x = data.node_features["feat"].float()
+        ys.append(data.labels)
+        y_hats.append(model(blocks, x))
+
+    res = MF.accuracy(
+        torch.cat(y_hats),
+        torch.cat(ys),
+        task="multiclass",
+        num_classes=num_classes,
+    )
+
+    return res
+
+
+def train(args, graph, features, train_set, valid_set, num_classes, model):
+    optimizer = torch.optim.Adam(model.parameters(), lr=args.lr)
+    dataloader = create_dataloader(
+        args, graph, features, train_set, is_train=True
+    )
+
+    for epoch in tqdm.trange(args.epochs):
+        # print(f"The beginning of epoch {epoch}")
+        model.train()
+        total_loss = 0
+        for step, data in tqdm.tqdm(enumerate(dataloader)):
+            # print(f"epoch: {epoch}, step: {step}")
+
+            # blocks = data.to_dgl_blocks()
+
+            # The input features from the source nodes in the first layer's
+            # computation graph.
+            # x = blocks[0].srcdata["feat"]
+            x = data.node_features["feat"].float()
```

Comment:
Instead of converting the data to float, better convert the model the designated data type.